### PR TITLE
Look ups

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: wellregistry/registry/migrations/0003_country_lookups.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""
```
Comment (mbucknell) [inline wellregistry/registry/migrations/0003_country_lookups.py:1]: @ayan-usgs , this was automatically generated, correct? Ed, you should be using the makemigration command to create the additional migrations that will be needed.

File: wellregistry/registry/models.py [hunk 2]
```diff
@@ -6,6 +6,20 @@
...
 
 
+class CountryLookup(models.Model):
+    """
+    Country lookup table for the Registry app.
+
+    Used to populate drop downs.
+
+    """
+    country_code = models.CharField(max_length=10)
```
Comment (mbucknell) [inline wellregistry/registry/models.py:16]: So this creates an artificial key. I think we should we be using the country_code? I think this is what the legacy NGWMN expects.

File: wellregistry/registry/migrations/0003_country_lookups.py (hunk 1 above)
Comment (ayan-usgs) [reply wellregistry/registry/migrations/0003_country_lookups.py:1]: Yep, they were automatically generated.

File: wellregistry/registry/models.py (hunk 2 above)
Comment (ayan-usgs) [reply wellregistry/registry/models.py:16]: Sounds good. I'll make the changes.

**Pull request merged**